images/10-virtualboxtools/run

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+kernel_dir=/lib/modules/`uname -r`
+if [ ! -d $kernel_dir/extra ];then
+    mkdir $kernel_dir/extra
+else
+    echo "Directory $kernel_dir/extra already exists."
+fi
+
+lsmod | grep "vboxsf" > /dev/null 2>&1
+if [ $? -ne 0 ];then
+    cp /dist/*.ko /lib/modules/`uname -r`/extra
+    depmod
+    modprobe vboxguest
+    modprobe vboxsf
+else
+    echo "Vbox module has been loaded."
+fi
+
+for _ in {1..10}
+do
+    if [ -f /run/console-done ];then
+        system-docker exec console ls /sbin/VBoxService /sbin/mount.vboxsf /bin/VBoxControl > /dev/null 2>&1
+        if [ $? -ne 0 ];then
```

Comment:
Do we need to add `else` block? Others situation should continue the loop until the number is exceeded